R/run.R

```diff
@@ -679,16 +679,19 @@ view_run_details <- function(run, auto_refresh = TRUE) {
 
     # stop and remove any existing widget job before submitting script
     # nolint start
-    if (exists("ACTIVE_WIDGET_JOB")) {
-      try(rstudioapi::jobSetState(ACTIVE_WIDGET_JOB, "succeeded"),
+    existing_job_id <- Sys.getenv("EXISTING_WIDGET_JOB")
```

Comment:
I'd give a azureml specific name, for example
EXISTING_WIDGET_JOB -> AZUREML_RSUDIO_WIDGET_JOB

Reply:
okay